mthree/mitigation.py

```diff
@@ -222,20 +144,35 @@ def cals_from_system(self, qubits=None, shots=None, method='balanced',
         Raises:
             M3Error: Called while a calibration currently in progress.
         """
+        from .calibration import grab_additional_cals
```

Comment:
Can you explain why this is here.

Reply:
In the code comments I mean

Reply:
This comes right after the comments. The goal here is to avoid the cyclic dependency `QuantumCircuit` -> `qiskit.result` -> `M3Mitigator` -> `QuantumCircuit`. This ensures `QuantumCircuit` will only be imported if calibration is explicitly called; and in the wrapper we do not use this capability (for any of the `qiskit-terra` mitigators).